src/View/User/Includes/_contact_grey_block.html.twig

```diff
@@ -0,0 +1,29 @@
+{% block contact_grey_block %}
+    <form class="contact-grey-title" methode="POST" novalidate>
+        <div class="grey-block col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+            <div class="contact-grey-section">
+                <div class="mb-3">
+                                    <label class="form-label">Nom</label>
+                    <input type="text" class="form-control" name="nom">
+                </div>
+                <div class="mb-3">
+                    <label class="form-label">Prenom</label>
```

Comment:
your labels are not linked to your inputs. use for = "xxx" in labels and id = "xxx" in entries